book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+2. A sophisticated algorithm that estimates the individual linear prediction, $\hat{\eta}$.
+
+These are composed as either:
 
-Predicting a survival distribution refers specifically to predicting the distribution of a subject's survival time, i.e., modelling the distribution of the event occurring over $\NNReals$.
-Therefore, this is seen as the probabilistic analogue to the deterministic time-to-event prediction.
+1. $\hatbh(t)\exp(\hat{\eta})$ if proportional hazards is assumed; or
+2. $\hatbh(\exp(-\hat{\eta})t)\exp(-\hat{\eta})$ for accelerated failure time models.
 
-Distributional prediction can, in theory, target any of the quantities introduced in @sec-distributions, but predicting $S(t)$ and/or $h(t)$ is most common.
-Hazard based approaches are particularly relevant for non- and semi-parametric estimation of the distribution, where no (or few) assumptions are made about the underlying distribution of event times.
+## Beyond single-event
 
-As mentioned above, all prediction types can theoretically be derived from a survival distribution prediction.
-For example, a time-to-event prediction can be obtained via $E(Y|\xx) = \int_0^\infty \hat{S}(t)$.
-However, for non-parametric methods the estimated cdf is often improper in the presense of censoring and thus integration requires extrapolation of the cdf [@Sonabend2022].
-For parametric models, the distribution of event times is fully specified once the paramers of the assumed distribution have been estimated, however, if the parameters were estimated based on only a small subset of the possible domain of $Y$, this essentially still constitutes extrapolation and will in most cases yield implausible predictions.
-A popular alternative is therefore to estimate the *restricted mean survival time* (RMST; @han.restricted.2022; @andersen.regression.2004).
+In @sec-eha, competing risks and multi-state models were introduced with a focus on estimating probability distributions.
 
-Predicting survival distributions is a type of *probabilistic survival task*, defined by predicting a conditional distribution over the positive Reals, $g: \calX \rightarrow \calS$ where $\calS \subseteq \Distr(\NNReals)$ is a convex set of distributions on $\NNReals$.
+In the competing risks setting, 'survival time' is ill-defined, it is ambiguous whether this refers to the time until a specific event or until any event takes place.
+In principle, one could apply the RMSE approach to the all-cause survival function, though this is uncommon in practice.
+Analogous to the single-event setting, a cause-specific prognostic index can be derived from a cause-specific cumulative incidence function, and vice versa.
+One might also consider transformations based on all-cause risk, but again such approaches are rarely used in applied settings.
 
-<!-- FIXME: Example + Illustration -->
+In multi-state models, the primary prediction targets are transition probabilities between states over time.
```

Comment:
primary prediction targest are transition hazards (ref) and transition probabilities (ref)...